views/tenants/show.erb

```diff
@@ -0,0 +1,3 @@
+<h2><%= @tenant.name %> <a href="/tenants/<%= @tenant.id %>/edit">(edit)</a></h2>
+ 
+Apartment: <%= @tenant.name %>
```

Comment:
Interesting.  Labeling the tenant.name as "Apartment".  I would expect to see more information about this tenant on the show page.